interpreter/cling/include/cling/Interpreter/DynamicLibraryManager.h

```diff
@@ -97,6 +97,9 @@ namespace cling {
   public:
     DynamicLibraryManager(const InvocationOptions& Opts);
     ~DynamicLibraryManager();
+    DynamicLibraryManager(const DynamicLibraryManager&) = delete;
```

Comment:
Can we now make `m_Dyld` a `unique_ptr`, please?

Reply:
No, we can't. The default constructor needs to be in the same file which is inconvenient (requires quite a few more includes).

Reply:
I see no default ctor declared, and the compiler won't generate one as others are declared. There are several ctors but those are all outlined.

Reply:
I meant the outlined ctor we have needs to be moved here.

Reply:
O right. Yes, that sounds like an improvement!

Reply:
O but you were saying it's inconvenient, right, unfortunate :-( Fair enough, I will keep forgetting why we cannot use unique_ptr until you reactor this ;-) Certainly good enough for now, thanks for explaining!